### ISSUE TITLE
ModuleNotFoundError: No module named 'segmentation_models_pytorch.decoders'

### ISSUE THREAD

**Issue by zabir-nabil** [qubvel/segmentation_models.pytorch#530]
Something is probably broken in the latest version. I am getting the following error:
```
ModuleNotFoundError: No module named 'segmentation_models_pytorch.decoders'
```

Slightly older version works,
`pip install -U segmentation-models-pytorch==0.2.0`

**Comment by ssnirgudkar** [comment]
0.2.0 and 0.2.1 work technically, but i think there's some issue with losses. Training loss is extremely less, but loss during inferences are extremely poor. It's as if loss and IOU have been swapped.

**Comment by zabir-nabil** [comment]
`0.2.1` doesn't work, at least not on my windows.

**Comment by qubvel** [comment]
Should be fixed with the latest PR #531
If you still have some issues, please, try to uninstall `pip uninstall segmentation-models-pytorch` and then install the package again
